### PR TITLE
Api/validator

### PR DESCRIPTION
**Pull request by ovekyc**
Add item-api validation module
and Edit package.json name

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
-  "name": "example-project",
+  "name": "project-goober",
```
Comment (festinalente91) [inline package.json:2]: finally :)

File: src/server/routes/api-routes.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import express from 'express';
 import items from '../items/routes';
+import validator from '../items/validator';
 
 const router = express.Router();
 
-router.use('/items', items);
+router.use('/items', validator, items);
```
Comment (festinalente91) [inline src/server/routes/api-routes.js:7]: just for the record, 
we might be able to drag this validator out to  outer router.

**Review by festinalente91 — COMMENTED** [review]
LGTM :)

File: package.json (hunk 1 above)
Comment (mun625) [reply package.json:2]: yeah!

Comment (shberry16) [reply package.json:2]: nice!

File: src/server/items/routes.js [hunk 3]
```diff
@@ -246,3 +245,4 @@ router.put('/:id', ItemContollers.modify);
 router.delete('/:id', ItemContollers.remove);
 
 export default router;
+
```
Comment (shberry16) [inline src/server/items/routes.js]: Is there a reason you added this extra line? If not, please remove :)

Comment (ovekyc) [reply src/server/items/routes.js]: removed it! Thank you Soo!

**Comment by festinalente91** [comment]
lgtm

**Comment by ovekyc** [comment]
lgtm in army? What a Patrick lol

**Comment by hewonjeong** [comment]
LGTM

**Pull request merged**